tasks/build/download_node_builds.js

```diff
@@ -1,84 +1,133 @@
-module.exports = function (grunt) {
-  let { map, fromNode } = require('bluebird');
-  let { resolve } = require('path');
-  let { pluck } = require('lodash');
-  let { createWriteStream } = require('fs');
-  let { createGunzip } = require('zlib');
-  let { Extract } = require('tar');
-  let { rename } = require('fs');
-  let wreck = require('wreck');
-
-  let platforms = grunt.config.get('platforms');
-  let activeDownloads = [];
-
-  let start = async (platform) => {
-    let finalDir = platform.nodeDir;
-    let downloadDir = `${finalDir}.temp`;
+import { Promise, map, fromNode, promisify } from 'bluebird';
+import { resolve, basename, dirname } from 'path';
+import { createReadStream, createWriteStream, writeFile } from 'fs';
+import { createGunzip } from 'zlib';
+import { Extract } from 'tar';
+import { rename } from 'fs';
+import checkHash  from 'check-hash';
+import wreck from 'wreck';
 
-    if (grunt.file.isDir(platform.nodeDir)) {
-      grunt.log.ok(`${platform.name} exists`);
-      return;
-    }
+const wreckGet = promisify(wreck.get, wreck);
+const checkHashFromFile = promisify(checkHash.fromFile);
```

Comment:
Might as well make these top to promisifys have consistent naming with the next two, so use the `Async` suffix.